src/icons/Checkbox.js

```diff
@@ -1,43 +1,45 @@
 import React from 'react'
+import { iconPropType } from '../common-prop-types'
+import { styles } from './styles.js'
 
 export function Unchecked({ className }) {
     return (
         <svg
             xmlns="http://www.w3.org/2000/svg"
-            width="48"
-            height="48"
             viewBox="0 0 48 48"
             className={className}
         >
             <path d="M38 10v28H10V10h28m0-4H10c-2.21 0-4 1.79-4 4v28c0 2.21 1.79 4 4 4h28c2.21 0 4-1.79 4-4V10c0-2.21-1.79-4-4-4z" />
+            <style jsx>{styles}</style>
         </svg>
     )
 }
+Unchecked.propTypes = iconPropType
 
 export function Checked({ className }) {
     return (
         <svg
             xmlns="http://www.w3.org/2000/svg"
-            width="48"
-            height="48"
             viewBox="0 0 48 48"
             className={className}
         >
             <path d="M38 6H10c-2.21 0-4 1.79-4 4v28c0 2.21 1.79 4 4 4h28c2.21 0 4-1.79 4-4V10c0-2.21-1.79-4-4-4zM20 34L10 24l2.83-2.83L20 28.34l15.17-15.17L38 16 20 34z" />
+            <style jsx>{styles}</style>
         </svg>
     )
 }
+Checked.propTypes = iconPropType
 
 export function Indeterminate({ className }) {
```

Comment:
This is broken with the refactor: 
![image](https://user-images.githubusercontent.com/185449/63796593-a531c580-c906-11e9-9159-c9ff32dc14b1.png)

Reply:
I fixed this now, but not quite as I would have liked... For all these icons I've been getting the production svg at 48px, however for `Indeterminate` [this seems to be unavailable](https://github.com/google/material-design-icons/tree/master/toggle/svg/production). So I've had to resort to putting `viewBox="0 0 24 24"`.